app/scripts/constants.js

```diff
@@ -605,13 +605,16 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
     'icon-cordova': 'cordova.png',
     'icon-datagrid': 'datagrid.svg',
     'icon-datavirt': 'datavirt.svg',
+    'icon-debian': 'debian.svg',
     'icon-decisionserver': 'decisionserver.svg',
     'icon-django': 'django.svg',
     'icon-dotnet': 'dotnet.svg',
     'icon-drupal': 'drupal.svg',
     'icon-eap': 'eap.svg',
     'icon-elastic': 'elastic.svg',
     'icon-erlang': 'erlang.svg',
+    'icon-fedora': 'fedora.svg',
+    'icon-free-bsd': 'free-bsd.svg',
```

Comment:
I noticed when looking for a single-color FreeBSD logo that per https://www.freebsdfoundation.org/about/project/, FreeBSD is one word.  Mind changing "icon-free-bsd" to "icon-freebsd" and "free-bsd.svg" to "freebsd.svg" and rename the actual image as well?  I used "icon-freebsd" in https://github.com/openshift/openshift-logos-icon/pull/16, and the class names will need to match.

While making that change, can you also squash to a single commit?

Thanks, @rthallisey.